### ISSUE TITLE
Integrate More Leaky Keys

### ISSUE THREAD

**Issue by pdparchitect** [zricethezav/gitleaks#200]
A great resource of various types of sensitive keys can be found here: https://gist.github.com/EdOverflow/8bd2faad513626c413b8fc6e9d955669

**Comment by zricethezav** [comment]
Appreciate this @pdparchitect! I plan to add a couple more by default but then maybe add an `--expanded` option for users who want an expanded search

**Comment by zricethezav** [comment]
exporting these into a valid gitlieaks config file in the examples directory would be a good first issue for someone

**Comment by stone-z** [comment]
Hey @zricethezav I would be open to picking this up. It would be helpful to me to include [these patterns](https://github.com/dxa4481/truffleHogRegexes/blob/master/truffleHogRegexes/regexes.json) (or at least those which aren't already included) by default, you also mentioned you had some others you'd like to include. I can go ahead and add them if you're open to it

**Comment by zricethezav** [comment]
@stone-z adding the regexes that are not already included from the list you provided would be a nice addition. I was hoping to add a gitleaks.toml config file that included all the regexes that @pdparchitect provided in their link. This config would live in the `/examples` directory of this project.

**Comment by dhra** [comment]
Please check [this resources](https://github.com/Shashank-In/TravisLeaks/blob/master/wordlist.txt)

**Comment by stone-z** [comment]
@zricethezav I found that the current default patterns included the majority of the trufflehog patterns. I cleaned up some issues I found in https://github.com/zricethezav/gitleaks/pull/356, which also expands the generic credential pattern to include almost everything @pdparchitect and @dhra suggested.
Neither of those lists included (new) patterns for the keys themselves, only for the variable names. Most of these are now matched by the generic credential pattern.
